### DISCUSSION TITLE
Render component on top of the bottom sheet

### DISCUSSION THREAD

**Discussion by falk-stefan** [gorhom/react-native-bottom-sheet#1835] [Q&A] [answered]
The following example from the repository shows a small icon which displays some weather information (Cloudy, 12°):

![image](https://github.com/gorhom/react-native-bottom-sheet/assets/58136779/361b42c4-a5b4-4886-a396-7f26eb36a506)

What is the recommended way to render something on top of the bottom sheet? I couldn't find a straight forward way other than render my own handle-component so far.

**Comment by kenniford** [comment]
hello, were you able to figure this out?

**Reply by falk-stefan** [reply, marked as answer]
Yes. You can use `handleComponent` and customize it:

```html
<BottomSheet
        ref={sheetRef}
        index={index}
        handleComponent={() => <HandleComponent onMinimizePressed={handleMinimizeButtonPush} />}
        >
</BottomSheet>
```

**Reply by aeben-mtn-gh** [reply]
To get it outside the component, you can do something like; 

` handleComponent={() => (
          <Body style={{ top: -40, backgroundColor: "pink" }} text="Hell0" />
        )}`

**Comment by samliok** [comment]
How can you do the weather icon in the example? it is placed outside the themed bottom sheet, yet moves with it.

**Reply by falk-stefan** [reply]
I solved it by creating a custom handle component [as explained here](https://github.com/gorhom/react-native-bottom-sheet/discussions/1835#discussioncomment-10399963).
